lang/cs/Org.Apache.REEF.Client/Avro/YARN/AvroYarnJobSubmissionParameters.cs

```diff
@@ -60,6 +60,12 @@ public static string Schema
         public string jobSubmissionDirectoryPrefix { get; set; }
 
         /// <summary>
+        /// Gets or sets file system url
```

Comment:
This needs a bit more explanation. What is the file system URL? Do you mean the URL of the namenode?